### DISCUSSION TITLE
Is there any plan on moving "@emotion/react" to peer dependency ?

### DISCUSSION THREAD

**Discussion by mohitarora249** [JedWatson/react-select#5095] [Q&A] [answered]
Is there any plan on moving "@emotion/react" to peer dependency ?

**Comment by Methuselah96** [comment, marked as answer]
Yes, the plan is detailed in https://github.com/JedWatson/react-select/discussions/4858. We do not plan to make `@emotion/react` a `peerDependency` until we split `react-select` into separate packages to make sure users don't get stuck in a corner. However there's no timeline of when that will happen.
